sequential-convoy/manifest.json

```
@@ -0,0 +1,60 @@
+{
+    "title": "Service Bus Peek Lock Pattern - Renew lock token",

```

Consistency with other templates:

```suggestion
    "title": "Azure Service Bus: Get message from a queue with token renewal - Peek lock renew pattern",

```